### ISSUE TITLE
Y24-265 - New LIMS pipeline for FFPE RNA sample processing

### ISSUE THREAD

**Issue by SujitDey2022** [sanger/limber#1861]
**User story**
As a Short Read Team Lead (Liz E.) I want LIMS integration for the FFPE RNA pipeline SO THAT samples and outcomes from this pipeline are better tracked enabling quality and provisioning cross charging data.

**Details:**

This new pipeline will be a part of the RNA pipeline and will be a Single Library only submission for reISC. It will be similar to RNA-A submissions and will be called RNA-F. The plate purposes will be as indicated in the arrow diagram below. 

![image](https://github.com/user-attachments/assets/e4640ef2-dd8f-4fa9-b146-3616cdf9c8b9)

There are no additional plates in this process, but there will be a new library type RNA -FFPE with submission template RNA-F.

**Who are the primary contacts for this story**
Liz E.

**Who is the nominated tester for UAT**
Liz E.

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] New Submission Type
-  New Library Type: RNA-FFPE _**(May need to inform NPG about this)**_
- [ ] Bed verification is in place
- Bed Verification details
 -- Machine - Bravo
![image](https://github.com/user-attachments/assets/fae13440-2c2f-4673-9040-af216565f434)
- [ ] Printing is in place

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by ElizabethEasthope** [comment]
<img width="430" alt="RNA-FFPE" src="https://github.com/user-attachments/assets/24b57bc0-97c5-47da-b766-c2f15f0d0878">

Here is an image of the limber flow for the FFPE RNA process.  Thanks Liz

**Comment by ElizabethEasthope** [comment]
Here is an image of the plate purposes and beds for verification.
<img width="288" alt="bed verification for FFPE" src="https://github.com/user-attachments/assets/db916e70-a530-41b4-b91d-bbb16f107c3a">

**Comment by ElizabethEasthope** [comment]
The bed verification is all for the Bravo instrument.  
The plate purposes will be as indicated in the arrow diagram above.  There are no additional plates in this process, but there will be a new library type RNA -FFPE with submission template RNA-F.
No other acceptance criteria is needed other than those already discussed in meeting on 21st August (Sujits notes)

**Comment by SujitDey2022** [comment]
@ElizabethEasthope please can you confirm if the acceptance criteria covers all the discussed requirements.

**Comment by andrewsparkes** [comment]
@ElizabethEasthope please can you clarify if the step from the Cherrypick to the LBR Frag is a straight stamp of all wells or will this be a partial submission and transfer? All the other pipeline variants in the diagram seem to be straight stamps in all steps, but just wanted to check. Thx.

**Comment by ElizabethEasthope** [comment]
@andrewsparkes Hi Andrew, Yes the Cherry-Pick to LBR Frag is a straight stamp of samples without any sample relocation.  There will be straight stamps along the whole process for this process too.  Thanks Liz

**Comment by ElizabethEasthope** [comment]
@SujitDey2022 I have looked through the criteria and all seem to be correct for me.  Thanks Liz

**Comment by BenTopping** [comment]
Temporarily `On Hold` waiting for stakeholder confirmation of purposes.
